modules/appeals_api/app/serializers/appeals_api/higher_level_review_serializer.rb

```diff
@@ -4,6 +4,7 @@ class AppealsApi::HigherLevelReviewSerializer
   include FastJsonapi::ObjectSerializer
   set_key_transform :camel_lower
   attributes :status, :updated_at, :created_at
-  attribute :form_data, if: proc { |_record, options| !options[:is_collection] }
+  # only return form_data for created records
+  attribute :form_data, if: proc { |record| record.saved_change_to_id? }
```

Comment:
What do ya'll thing, before we passed in a :is_collection flag, but, now, since "create" is the only controller action where we return the form_data back to the caller, I changed the serializer to "detect" changes to the records ID, which only happens on create.
Would it be better to pass in an explicit option instead, like the old :is_collection..idk :return_form_data ?

Reply:
I'm totally fine with asking the record for its newness in the serializer - no need to pass down options.